UsageDocs/Installing SwiftDiscord for iOS.md

```diff
@@ -103,12 +104,10 @@ You may be tempted to skip some steps or to not follow them in order. If you'd l
 	        }
 	    }
 	    
-	    
-	    func client(_ client: DiscordClient, needsVoiceEncoderForEngine engine: DiscordVoiceEngine) throws -> DiscordVoiceEncoder {
-	        return DiscordVoiceEncoder(opusEncoder: try DiscordOpusEncoder(bitrate: 128_000, sampleRate: 48_000, channels: 2))
+	    func client(_ client: DiscordClient, needsDataSourceForEngine engine: DiscordVoiceEngine) throws -> DiscordVoiceDataSource {
+	        return self as! DiscordVoiceDataSource
```

Comment:
Did you test this (with audio)?  I don't think this will go well, since I don't see the view controller conforming to DiscordVoiceDataSource anywhere.
The new equivalent to the old code is
```swift
return DiscordBufferedVoiceDataSource(
    opusEncoder: DiscordOpusEncoder(bitrate: 128_000, sampleRate: 48_000, channels: 2)
)
```

Reply:
You are probably right. My goal in fixing the sample code was to get it to a point where people can easily get the code to compile and run and not necessarily concern itself with setting up various features with the framework. I believe that sample code to get other features up and running should be provided elsewhere and not in the basic project setup guide.

For my use case, I want to get the project compiling and running just fine with the simulator working so that I can get a git commit in. I am not relying on the sample code to contain advanced functionality. I changed the sample code to something that can be pasted in and compile without issue as I noticed that the code you had there initially caused the compiler (on Xcode 9 and iOS 11) to give compilation errors that could mislead folks into thinking it was an issue with project setup when in reality it was a syntax issue.